### PR TITLE
Added deprecated message for fields that should source from SSO

### PR DESCRIPTION
**Pull request by webbyfox**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

**Review by depsiatwal — COMMENTED** [review]
CL ?

**Comment by webbyfox** [comment]
Added @depsiatwal

File: company/models.py [hunk 1]
```diff
@@ -174,18 +174,20 @@ def save(self, *args, **kwargs):
...
+    # Deprecated, Email field should be used from SSO.User.Email
     company_email = models.EmailField('company email', unique=True)
     is_active = models.BooleanField(
         verbose_name='active',
         default=True,
         help_text='Unselect this instead of deleting accounts.',
     )
     date_joined = models.DateTimeField(verbose_name='date joined', default=timezone.now,)
-    # deprecated in favour of company.models.Company.contact_details
+    # Deprecated in favour of company.models.Company.contact_details
```
Comment (depsiatwal) [inline company/models.py:190]: wouldn't this be SSO contract details ?

Comment (webbyfox) [reply company/models.py:190]: yes, the hence deprecated message above

**Review by depsiatwal — APPROVED** [review]

**Pull request merged**